lib/index.js

```diff
@@ -1,29 +1,68 @@
 'use strict';
 
+var pg = require('pg');
 var addDefaultValues = require('./add_default_values.js');
 var loadSchema = require('./load_schema.js');
 var updateSchema = require('./update_schema.js');
+var routeGen = require('./helpers/endpoints_helpers/endpoints_generator.js');
+var configurationRoutes = require('./helpers/configuration/configuration.js');
+var db = require('./db.js');
+
+require('env2')('config.env');
 
 module.exports = function (server, options, next) {
+  var routes;
+
+  if (!process.env.DATABASE_URL) {
```

Comment:
Could we add the DATABASE_URL in the options of the plugin instead of using an environment variable, I think it's more consistent to have all the settings of the plugin in the options object